package.json

```diff
@@ -26,15 +26,23 @@
     "dev:ct:helpers": "cd packages/commercetools/helpers && yarn dev",
     "dev:ay:api-client": "cd packages/about-you/api-client && yarn dev",
     "dev:ct": "cd packages/commercetools/theme && yarn dev",
-    "heroku-postbuild": "yarn build:ct"
+    "heroku-postbuild": "yarn build:ct",
```

Comment:
can you delete that? and the Procfile - we are not using it anymore

Reply:
sure, on my way